src/lib/module-helpers.ts

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { PeprValidateRequest } from "./validate-request";
+import { a } from "../lib";
+
+// Returns all containers in the pod
+export function containers(
```

Comment:
I think `GetContainers` would align closer to the current helper functions

Reply:
similar to SetLabel, HasLabel, SetAnnotation, HasAnnotation

Reply:
In looking at the below-linked uds-core section, would it make sense to update the function to something like:

export function GetContainers(
  request: PeprValidateRequest<a.Pod> | PeprMutateRequest<a.Pod>,
  containerTypes?: "containers" | "initContainers" | "ephemeralContainers",
)

Reply:
We considered a couple of different ways to access the GetContainers function and it didn't seem especially tied to validation -- seemed more like a selector function (since it just looks up a set of container keys in a K8s.Pod object). By keeping it in a separate module-helpers file, we can also use it for mutate requests as well by updating the request parameter to PeprValidateRequest<a.Pod> | PeprMutateRequest<a.Pod>.

Reply:
That is a great point

Reply:
since it is not going to be a part of the fluent api in the same way i feel less strongly about the renaming